app/controllers/idv/doc_auth_controller.rb

```diff
@@ -24,6 +26,12 @@ def redirect_if_pending_profile
       redirect_to verify_account_url if current_user.decorate.pending_profile_requires_verification?
     end
 
+    def update_if_skipping_upload
+      return unless params[:step] == 'upload' && flow_session&.[](:skip_upload_step)
```

Comment:
I really dislike `unless` because of readability, especially with a compound statement. WDYT of flipping it?

```suggestion
      return if params[:step] != 'upload' || !flow_session || !flow_session[:skip_upload_step]
```

I *think* I got this right and I think we an expand out the `&.[]` which always looks goofy to me for a possibly-nil hash

Reply:
> I really dislike `unless` because of readability, especially with a compound statement. WDYT of flipping it?

Sounds good to me 👍